.gitignore

```diff
@@ -40,3 +40,4 @@ android/
 changed.txt
 eslint-report.json
 tsc-report.json
+.env
```

Comment:
@Danar435 I think you can remove this line as `.env.local` is the customary env file to use and it's already ignored `.env` is NOT ignored as it can be used to push certain values to production on purpose. `.env.local` is for LDE purposes.